lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -212,61 +212,94 @@
   }
 }
 
-// +Dashboard - Course Listing
-// ====================
-.dashboard {
-  .my-courses {
+.xseries-epmty-section{
```

Comment:
Actually, can it be more generic than that? .dashboard-empty-section or something?